callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -0,0 +1,114 @@
+package io.callstats.sdk;
+
+import java.io.IOException;
+
+import io.callstats.sdk.messages.BridgeEventMessage;
+import io.callstats.sdk.messages.BridgeEventResponse;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.ParseException;
+import org.apache.http.util.EntityUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.google.gson.Gson;
+
+/**
+ *
+ * @author Karthik Budigere
+ */
+public class CallStats {	
+	private CallStatsHttpClient httpClient;
+	private int appId;
+	private String appSecret;
+	private String bridgeId;
+	private CallStatsInitListener listener;
+	private CallStatsAuthenticator authenticator;
+	private static final Logger logger = LogManager.getLogger("CallStats");
+	private Gson gson;
+	private EndpointInfo endpointInfo;
+	private boolean isInitialized;
+			
+	public boolean isInitialized() {
+		return isInitialized;
+	}
+
+	private void setInitialized(boolean isInitialized) {
+		this.isInitialized = isInitialized;
+	}
+
+	public CallStats() {
+		gson = new Gson();
+	}
+
+	public void intialize(final int appId, String appSecret,final String bridgeId,EndpointInfo endpointInfo,CallStatsInitListener callStatsInitListener) {
```

Comment:
A general comment, use some style formatter (like the one comes with eclipse) to reformat the java files so that it looks nicer.